src/index.html

```diff
@@ -4,9 +4,42 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+
+    <main class="calendar calendar--month-length--29">
+      <div class="calendar__cell calendar__cell--1 calendar__cell--thu "></div>
```

Comment:
`--thu` shouldn't be here
it should be `start-day` in the calendar:
![image](https://user-images.githubusercontent.com/86923089/196486106-adf7957a-e81a-4440-8550-777bc31f5b38.png)